internal/project/core_test.go

```diff
@@ -53,3 +57,198 @@ func TestCore_ExistsWithID(t *testing.T) {
 	assert.True(t, ok)
 	assert.NoError(t, err)
 }
+
+func TestCore_Get(t *testing.T) {
+	type fields struct {
+		repo IRepo
+	}
+	type args struct {
+		ctx       context.Context
+		projectID string
+	}
+	ctrl := gomock.NewController(t)
+	mockRepo := mocks.NewMockIRepo(ctrl)
+	ctx := context.Background()
+
+	project1 := &Model{
+		Name:      "test1",
+		ProjectID: "testID1",
+		Labels:    map[string]string{"label": "value"},
+	}
+	project2 := &Model{
+		Name:      "test2",
+		ProjectID: "testID2",
+		Labels:    map[string]string{"label": "value"},
+	}
+	project3 := &Model{
+		Name:      "test3",
+		ProjectID: "testID3",
+		Labels:    map[string]string{"label": "value"},
+	}
+	project1.SetVersion("1")
+	project2.SetVersion("2")
+	project3.SetVersion("3")
+
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *Model
+		wantErr bool
+	}{
+		{
+			name: "Get Project 1 Successfully.",
+			fields: fields{
+				repo: mockRepo,
+			},
+			args: args{
+				ctx:       ctx,
+				projectID: project1.ProjectID,
+			},
+			want:    project1,
+			wantErr: false,
+		},
+		{
+			name: "Get Project 2 Successfully.",
+			fields: fields{
+				repo: mockRepo,
+			},
+			args: args{
+				ctx:       ctx,
+				projectID: project2.ProjectID,
+			},
+			want:    project2,
+			wantErr: false,
+		},
+		{
+			name: "Get Project 3 Successfully.",
```

Comment:
Why are we testing the same case thrice? Can we randomly pick a number between 1 and 3 and see if it picked the correct project ID?